test/config.json

```diff
@@ -45,7 +45,10 @@
     },
     "peers": {
         "enabled": true,
-        "list": [],
+        "list": [{
+            "ip": "127.0.0.1",
+            "port": 9999
+        }],
```

Comment:
Assuming this is required for tests to pass, i.e. peer is not removed on disconnect.